.buildkite/pipeline.yml

```diff
@@ -31,6 +31,13 @@ steps:
           cpu: "8"
           memory: "4G"
 
+      - label: ":building_construction: Run local build: MacOS"
```

Comment:
@pazone, what's the reason we build this? IIUC, the tests step does not use these artifacts? Or am i seeing this wrongly?